packages/contractkit/src/wrappers/LockedGold.ts

```diff
@@ -187,4 +196,24 @@ export class LockedGoldWrapper extends BaseWrapper<LockedGold> {
       withdrawals[0]
     )
   }
+
+  /**
+   * Retrieves AccountSlashed for epochNumber.
+   * @param epochNumber The epoch to retrieve AccountSlashed at.
+   */
+  async getAccountsSlashed(epochNumber: number): Promise<AccountSlashed[]> {
+    const events = await this.getPastEvents('AccountSlashed', {
+      fromBlock: await this.kit.getFirstBlockNumberForEpoch(epochNumber),
+      toBlock: await this.kit.getLastBlockNumberForEpoch(epochNumber),
+    })
+    return events.map(
```

Comment:
are we sure that `e.returnValues` are the correct types we expect? In particular, `BigNumber`? They are typed as `any`

Reply:
Fixed with `valueToBigNumber` here and elsewhere.